### DISCUSSION TITLE
Show open windows/doors

### DISCUSSION THREAD

**Discussion by yfaykya** [Blackymas/NSPanel_HA_Blueprint#2147] [Q&A]
Hi -
Is it possible to create a screen/widget that would show a count of open contact sensors? Very useful to see when leaving the house!
Thanks

**Comment by edwardtfn** [comment]
You can easily create a template sensor in your Home Assistant and then display that as a regular sensor in your panel.
Please let me know if you need help to create that sensor in HA. 😉

**Comment by yfaykya** [comment]
Thanks for suggestion. I was playing with that but ideally I would also be able to list all the open ones too rather than just a count. 
Am I making sense? :-)
